### DISCUSSION TITLE
How to work with Nuitka for version 8.x ?

### DISCUSSION THREAD

**Discussion by JiaPai12138** [dashingsoft/pyarmor#1141] [Q&A] [answered]
For version 7.7.4, i checked [this](https://pyarmor.readthedocs.io/en/v7.7.4/advanced.html?highlight=nuitka#work-with-nuitka)
Is there any workaround for version 8?

**Comment by jondy** [comment]
I haven't test it. 

The obfuscated scripts still could be taken as `.py` scripts with a runtime package.

**Reply by null-routed** [reply]
I tried packing a pyarmor-obfuscated code with Nuitka and I got
```RuntimeError: unauthorized use of script (1:871)```
When running the executable. Is this being looked into or not? I could open a new issue with more details.

**Reply by jondy** [reply]
> I tried packing a pyarmor-obfuscated code with Nuitka and I got
> `RuntimeError: unauthorized use of script (1:871)`
> When running the executable. Is this being looked into or not? I could open a new issue with more details.
@null-routed 

This could be disabled by this command:
```
pyarmor cfg restrict_module=0
pyarmor gen foo.py
```

**Comment by jondy** [comment, marked as answer]
Also refer to
https://pyarmor.readthedocs.io/en/latest/how-to/third-party.html
